### PR TITLE
Docs: Adds nvidia ecs variant to README

### PR DESCRIPTION
**Pull request by stockholmux**
Signed-off-by: Kyle J. Davis <kyledvs@amazon.com>

**Issue number:** n/a

**Description of changes:** Adds ecs nvidia variant to the readme 

**Testing done:** eyeballs only

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by etungsten — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Review by arnaldo2792 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ The following variants support EKS, as described above:
 The following variant supports ECS:
```
Comment (arnaldo2792) [inline README.md]:
```suggestion
The following variants supports ECS:
```

**Review by arnaldo2792 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -59,9 +59,10 @@ The following variants support EKS, as described above:
 - `aws-k8s-1.22-nvidia`
 - `aws-k8s-1.23-nvidia`
 
-The following variant supports ECS:
+The following variants supports ECS:
```
Comment (bcressey) [inline README.md]:
```suggestion
The following variants support ECS:
```

**Review by bcressey — COMMENTED** [review]
Can you rebase locally to squash the commits together?

**Comment by stockholmux** [comment]
@bcressey Done. 
@arnaldo2792 Can you re-review to unblock merging?

**Review by bcressey — APPROVED** [review]

**Review by arnaldo2792 — APPROVED** [review]

**Pull request merged**